site/en/docs/extensions/reference/storage/index.md

```diff
@@ -128,17 +174,17 @@ to this setting are immediately saved to sync storage by the options page and th
 ```js
 // In-page cache of the user's options
 const options = {};
+const optionsForm = document.getElementById("optionsForm");
 
 // Initialize the form with the user's option settings
-chrome.storage.sync.get('options', (data) => {
-  Object.assign(options, data.options);
-  optionsForm.debug.checked = Boolean(options.debug);
-});
+const data = await chrome.storage.sync.get("options");
+Object.assign(options, data.options);
+optionsForm.debug.checked = Boolean(options.debug);
 
 // Immediately persist options changes
-optionsForm.debug.addEventListener('change', (event) => {
+optionsForm.debug.addEventListener("change", (event) => {
   options.debug = event.target.checked;
-  chrome.storage.sync.set({options});
+  chrome.storage.sync.set({ options });
 });
 ```
```

Comment:
We should declare a `setDebugMode` function in the background.js code sample. Folks sometimes get confused when they see functions like this that aren't declared in the code sample and assume that it's a Chrome global or something. Best to eliminate potential confusion.

```js
function setDebugMode() { /* ... */ }
// or
const setDebugMode = () => { /* ... */ };

Reply:
Sounds good 👍